plugins/generator-1.14.4/forge-1.14.4/procedures/world_data_fluidat.java.ftl

```diff
@@ -0,0 +1 @@
+(world.getFluidState(new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})).getBlockState())
```

Comment:
Please add "/\*@BlockState\*/" at the beginning of this template, as well as make it return AIR in case the block isn't fluid. The same applies to the 1.15.2 variant of this procedure.

Reply:
The procedure already returns AIR if there's no fluid at the specified coords